<launch>
  <include file="$(find pose_instance_builder)/launch/pi_tracker.launch" />
  <!--
  <node name="pose_labeler" 
        pkg="pose_labeler" type="pose_labeler_node.py" 
        required="true" output="screen">
        <param name='min_confidence' type="double" value="0.2"/>
  </node> 
  -->

  <node name="instance_builder"
        pkg="pose_instance_builder"
        type="instance_builder_node.py">
        <param name="builder_type" 
               value="instance_builder.PiTrackerIBuilder" type="str" />
        <param name="skeleton_topic" value="/skeleton" type="str" />
  </node>

</launch>
